### DISCUSSION TITLE
MPCM Mode?  Multi Processor Communication Mode

### DISCUSSION THREAD

**Discussion by Eddiiie** [SpenceKonde/ATTinyCore#821] [Support / Q & A]
Thank you for all the work you have done.  Unbelievable.

I am working on a project where I need to be a serial 9bit slave.      I came across this link that shows one (?) of the ATTINY models supports MPCM Mode, which is exactly what I am looking for.   
   Link - [https://www.avrfreaks.net/s/topic/a5C3l000000UcdaEAC/t162314](https://www.avrfreaks.net/s/topic/a5C3l000000UcdaEAC/t162314)

Forgive my laziness here, I've spent days going through datasheets, looking for libraries, replacing core code, on numerous other micros over the last days and am exhausted.  

I'd really like to use ATTINY something for my project.  

My question is -   Has anyone enabled this mode?   Is it available in all ATTINY chips (probably not an easy answer), is it in the core here, and how do I use it?  :)

Basically, if the UART (?) see's the 9th bit on the incoming frame, it knows that it is an address frame and will compare that address to a stored address you put in a register somewhere, if it matches, the frame is forwarded to the CPU.   If not, the UART ignores it (for how long?)..     it saves CPU time!!

Thank you!
